ProcessLib/BoundaryCondition/NeumannBoundaryConditionLocalAssembler.h

```diff
@@ -58,24 +64,36 @@ class NeumannBoundaryConditionLocalAssembler final
         // Get element nodes for the interpolation from nodes to integration
         // point.
         NodalVectorType parameter_node_values =
-            _neumann_bc_parameter.getNodalValuesOnElement(Base::_element, t)
+            _data.neumann_bc_parameter
+                .getNodalValuesOnElement(Base::_element, t)
                 .template topRows<ShapeFunction::MeshElement::n_all_nodes>();
 
+        ParameterLib::SpatialPosition position;
+        position.setElementID(Base::_element.getID());
+
+        double integral_measure = 1.0;
         for (unsigned ip = 0; ip < n_integration_points; ip++)
         {
+            position.setIntegrationPoint(ip);
             auto const& ip_data = Base::_ns_and_weights[ip];
+            auto const& N = ip_data.N;
+            auto const& w = ip_data.weight;
 
-            _local_rhs.noalias() += ip_data.N *
-                                    parameter_node_values.dot(ip_data.N) *
-                                    ip_data.weight;
+            if (_data.integral_measure)
+            {
+                integral_measure = (*_data.integral_measure)(t, position)[0];
```

Comment:
```suggestion
               const double  integral_measure = (_data.integral_measure) ? (*_data.integral_measure)(t, position)[0] : 1.0;
```

Reply:
I think this doesn't work. See the argument in the RobinBoundaryConditionLocalAssembler case.

Reply:
I see.